### PR TITLE
chore: revise the template repository

### PR DESCRIPTION
**Pull request by maximousblk**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: 🚀 Feature Request/Idea
 about: Suggest a feature or idea for this project
-labels: 'type: feature, status: needs triage'
+labels: 'feature request, needs triage'
```
Comment (oganexon) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]: We should unify repo labels on each repo at some point

File: .github/workflows/chore.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: 'Chores'
```
Comment (oganexon) [inline .github/workflows/chore.yml:1]: Good idea

**Review by oganexon — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (maximousblk) [reply .github/ISSUE_TEMPLATE/feature_request.md:4]: yeah I updated default labels for new repos but will have to update existing repos manually.

File: .github/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
-### Our Pledge
+## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
 contributors and maintainers pledge to making participation in our project and
 our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of experience,
-nationality, personal appearance, race, religion, or sexual identity and
-orientation.
+size, disability, ethnicity, sex characteristics, gender identity and expression,
```
Comment (t8) [inline .github/CODE_OF_CONDUCT.md:8]: I *think* `sex characteristics`, `gender identity`, and `sexual identity` are all very similar. I could be wrong though?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Template
```
Comment (t8) [inline README.md:1]: SO glad this is simplified.

File: .github/CODE_OF_CONDUCT.md (hunk 3 above)
Comment (maximousblk) [reply .github/CODE_OF_CONDUCT.md:8]: I didn't write this. I just replaced the old one with what GitHub provides/suggests

Comment (t8) [reply .github/CODE_OF_CONDUCT.md:8]: Ahh, okay. They know more than me!

**Review by t8 — APPROVED** [review]

**Pull request merged**